tests/test_activity_recognition_feedback.py

```diff
@@ -180,7 +180,7 @@ def test_get_score_feedback(
         10, 10, 10, par_interface, session_id, test_id, protocol_constant
     )
     df_score = feedback.get_feedback(0, list(range(10)), feedback_image_ids)
-    assert df_score[1][0] == 0.1987951807228915
+    assert df_score[1][0] == 0.1987951807228916
```

Comment:
Did you mean to change this? It would be odd if this changed as a result of switching build systems.

Reply:
Yeah, this started showing up as a test failure on my machine. I think this is floating point error, I think it would be better to use `isclose` than changing these values. I have created https://github.com/darpa-sail-on/sail-on-client/issues/137 to track this.